requirements/requirements.in

```diff
@@ -39,6 +39,7 @@ social-auth-core==3.2.0  # see UPGRADE BLOCKERs
 social-auth-app-django==3.1.0  # see UPGRADE BLOCKERs
 requests
 requests-futures  # see library notes
+sdb
```

Comment:
Is this necessary?  I thought we have this in `requirements-dev` or something.

Reply:
Good point.  Updating that now.